src/api/spec/factories/configuration.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :configuration do
```

Comment:
Otherwise I need to manually setup the attributes in the test. But I don't have strong feelings here, if you don't want the factory I can remove it and do it manually in the test...